### ISSUE TITLE
Job checking status failed for dev `618858406e13c7816f28c300`

### ISSUE THREAD

**Issue by jonrkarr** [reproducible-biomedical-modeling/CRBM-Viz#3539]
```
"statusReason": "Checking status for job with id 3575669 for simulation 618858406e13c7816f28c300: Status is FAILED",
```

**Comment by jonrkarr** [comment]
This is probably because errors in posting logs weren't being handled, which then prevents the failure reason from being posted.

**Comment by jonrkarr** [comment]
Fixed in #3544
